src/test/resources/org/jabref/logic/importer/fileformat/EndnoteXmlImporterTestArticle2.xml

```diff
@@ -77,8 +77,7 @@
                 <style face="normal" font="default" size="100%">ISI:000200148800007</style>
             </accession-num>
             <notes>
-                <style face="normal" font="default" size="100%">some notes
-                </style>
+                <style face="normal" font="default" size="100%">some notes</style>
```

Comment:
Not sure if we should modify the imported files. - Since you do a roundtrip test, is is OKish...

Reply:
Yes, I made sure the old case is also handled, though. We can go back to it if you wish. Initially I had thought it's by mistake.